Added Sesquicentennial footer bg image/svg text and styled

tomkrupka opened the pull request:
https://waynedotedu.tpondemand.com/entity/68220-program-the-sesquicentennial-footer

Looks like: https://app.hyfy.io/v/abBcD8b4f7n/

Still waiting on Design approval as well

nickdenardis commented on demo/index.html:
```diff
@@ -15,5 +15,12 @@
+
+<div class="wsu-sesquicentennial__background">
+    <div class="wsu-sesquicentennial">
+        <a class="wsu-sesquicentennial__link" href="//150.wayne.edu">
```
This can go directly to `https://` since the 150 site forces SSL and some of the sites this is on may not.

tomkrupka replied:
updated

nickdenardis commented on src/footer.scss:
```diff
@@ -0,0 +1,33 @@
+            padding: 30px;
+        }
+    }
+}
```
This is small, but can you make sure your editor is picking up the `.editorconfig`? Per https://github.com/waynestate/sesquicentennial-footer/blob/feature/sesquicentennial-footer/.editorconfig#L10 it should be inserting a newline at the end of this file.

tomkrupka replied:
added a new line

tomkrupka commented:
@nickdenardis I couldn't get axe extension to work locally.  But I did add the title tag in the svg.  Take a look plz

nickdenardis commented:
@tomkrupka 
Got it. I'm running it now and will make the small change.

tomkrupka commented:
@nickdenardis I had already added inside the <g> following this example:
https://developer.mozilla.org/en-US/docs/Web/SVG/Element/title

Would you like me to remove mine then?
Nm I just removed it ;-)

nickdenardis commented:
@tomkrupka Hmm.. Let me try again. It passed outside of the `<g>` tag.
